sys/targets/targets.go

```diff
@@ -791,6 +793,7 @@ func (target *Target) setCompiler(clang bool) {
 			target.CFlags = append(target.CFlags, "--target="+target.Triple)
 		}
 		target.CFlags = append(target.CFlags, "-ferror-limit=0")
+		target.CFlags = append(target.CFlags, "-no-pie")
```

Comment:
What we actually need is to disable PIE in the case `-static-pie` is passed. Maybe we need something like `processMergedFlags()` which will be applied right before calling the compiler, because `-pie` is added by individual tests.

Reply:
I think it could be reasonable to invoke `processMergedFlags` for these purposes.